### PR TITLE
Add Spectre button

### PR DESCRIPTION
**Pull request by cristianmoisa**
Screen Shot 2020-08-09 at 7 25 25 PM

### CODE REVIEW

File: pages/tests/[testsByRunId].tsx [hunk 1]
```diff
@@ -275,8 +307,17 @@ Tests.getInitialProps = async (context): Promise<Props> => {
...
   const tests = await testsByTestRunIdReq.json()
 
+  const getTestRunById = await fetch(
+    `${process.env.deltaCore}/api/v1/test_run/${testsByRunId}`,
+    {
+      method: "GET",
+    }
+  )
+  const data = await getTestRunById.json()
+
```
Comment (Jnegrier) [inline pages/tests/[testsByRunId].tsx:317]: This called my attention a lot, in theory, all the data about the test_run should be already on the first endpoint `/api/v1/tests_history/test_run/<test_run_id>`... so I took a look into the Core Service, and I saw that I didn't added `test_run_data` into that's the endpoint, apologies for this, will fix the Core service now. so you could remove this second call and use the data from the only one endpoint 😄

Comment (opishcheiko) [reply pages/tests/[testsByRunId].tsx:317]: oh, true! Sorry for that, Cristian
yesss, so there is no need to change the props everywhere above

File: pages/tests/[testsByRunId].tsx [hunk 2]
```diff
@@ -141,101 +145,121 @@ function Tests(props: Props) {
...
                     style={{
                       display: "flex",
                       marginTop: "20px",
-                      marginLeft: "300px",
+                      alignItems: "baseline",
+                      width: "50%"
                     }}
                   >
-                    <p
+                    {test_run_data.data.spectre_test_run_url ? (
```
[the remark this answers is not in the document]
Comment (opishcheiko) [reply pages/tests/[testsByRunId].tsx:160]: @Jnegrier , I think this condition is fine actually, it should check for the null as well existing . We have it around the code and it works good. Can be tested though to be sure

File: pages/tests/[testsByRunId].tsx [hunk 3]
```diff
@@ -67,20 +67,27 @@ type Props = {
   test_history: SuiteAndTest[]
 }
 
-function Tests(props: Props) {
-  const classes = useStyles(props)
+function Tests({ test_history }) {
+  const [history] = test_history
```
Comment (opishcheiko) [inline pages/tests/[testsByRunId].tsx:71]: Cristian, in theory all of this is redundant now and you can keep the old way of having props. test_run_data should be there already, inside props, so no need for this change (in theory)

Comment (cristianmoisa) [reply pages/tests/[testsByRunId].tsx:71]: I applied object/array destructuring there so we can write less.

Comment (opishcheiko) [reply pages/tests/[testsByRunId].tsx:71]: okay, good idea!
Maybe we'll need to put a comment there now, that `if you want to use any other test_history property, please add in to that array`

Comment (Jnegrier) [reply pages/tests/[testsByRunId].tsx:73]: Actually, if you load a page that still doesn't have tests, you will end up with `is not an iterable` error, I guess we could put some validation on it @cristianmoisa 

Screenshot 2020-08-10 at 12 07 45

**Pull request merged**